lib/myinfo.rb

```diff
@@ -30,9 +30,9 @@ def self.configure
   # Configuration to set various properties needed to use MyInfo
   class Configuration
     attr_accessor :singpass_eservice_id, :app_id, :client_id, :proxy, :private_key, :public_cert, :client_secret,
-                  :redirect_uri
+                  :redirect_uri, :gateway_key
```

Comment:
do we need add :gateway_url to attr_accessor? it's already in the attr_reader.. think we just need to add the one we need minimally.